back/spec/acceptance/users_spec.rb

```diff
@@ -23,11 +23,22 @@
         parameter :size, 'Number of users per page'
       end
       parameter :search, 'Filter by searching in first_name, last_name and email', required: false
-      parameter :sort, "Sort user by 'created_at', '-created_at', 'last_name', '-last_name', 'email', '-email', 'role', '-role'", required: false
+      parameter :sort, "Sort user by 'created_at', '-created_at', 'last_name', '-last_name', 'email', " \
+                       "'-email', 'role', '-role'", required: false
       parameter :group, 'Filter by group_id', required: false
-      parameter :can_moderate_project, 'Filter by users (and admins) who can moderate the project (by id)', required: false
-      parameter :can_moderate, 'Filter out admins and moderators', required: false
-
+      parameter :can_moderate, 'All admins + users with either a project &/or folder moderator role', required: false
+      parameter :can_moderate_project, 'All admins + users who can moderate the project (by project id), ' \
+                                       'excluding folder moderators of folder containing project ' \
+                                       '(who can, in fact, moderate the project), ' \
+                                       'OR Admins & all users with project moderator role ' \
+                                       '(if no project ID provided)', required: false
+      parameter :is_not_project_moderator, 'Users who are not admins, nor project moderator of project, ' \
+                                           'nor folder moderator of folder containing project (by project id), ' \
+                                           'OR Users who are not admins, nor have project moderator role ' \
+                                           '(if no project ID provided)', required: false
+      parameter :is_not_folder_moderator, 'Users who are not admins, nor folder moderator of folder (by folder id), ' \
+                                          'OR Users who are not admins, nor have folder moderator role ' \
+                                          '(if no folder ID provided)', required: false
```

Comment:
I took the opportunity to add these, as I overlooked this in #4432

Reply:
It would also make sense to add at least one test for `is_not_project_moderator` and `is_not_folder_moderator`, but maybe the next time.

Reply:
You mean the related scopes? (already in the spec file), or the actual endpoints (acceptance tests)?

Reply:
The actual endpoints. Because it combines two scopes ` @users.not_admin.and(@users.not_project_folder_moderator(`